### ISSUE TITLE
Reformation Day is doubled in some states of Germany

### ISSUE THREAD

**Issue by arty-name** [commenthol/date-holidays#16]
```javascript
const Holiday = require('./lib/index')
const public = new Holiday('DE', 'mv')
  .getHolidays(2017)
  .filter(h => h.type == 'public')
  .filter(h => h.date.match(/2017-10-31/));
console.log(public)
```
produces following output: 
```
[ { date: '2017-10-31 00:00:00',
    start: Tue Oct 31 2017 00:00:00 GMT+0100 (CET),
    end: Wed Nov 01 2017 00:00:00 GMT+0100 (CET),
    name: 'Reformationstag',
    type: 'public' },
  { date: '2017-10-31 00:00:00',
    start: Tue Oct 31 2017 00:00:00 GMT+0100 (CET),
    end: Wed Nov 01 2017 00:00:00 GMT+0100 (CET),
    name: 'Reformationstag',
    type: 'public' } ]
```

I guess it has something to do with it being celebrated as the country-wide holiday in 2017 only

**Comment by commenthol** [comment]
Is fixed with #19

**Comment by arty-name** [comment]
Thanks!
